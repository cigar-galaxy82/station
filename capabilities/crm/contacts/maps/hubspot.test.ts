import { SuperfaceTest } from '@superfaceai/testing-lib';

describe(`crm/contacts/hubspot`, () => {
  let superface: SuperfaceTest;

  beforeEach(() => {
    superface = new SuperfaceTest();
  });

  describe('Create', () => {
    it('should perform successfully', async () => {
      await expect(
        superface.run({
          profile: 'crm/contacts',
          provider: 'hubspot',
          useCase: 'Create',
          input: {
            email: 'test@example.com',
            firstName: 'Test',
            lastName: 'User',
            company: 'Station Test',
            country: 'USA',
            customProperties: {
              myproperty: 'value',
            },
          },
        })
      ).resolves.toMatchSnapshot();
    });
  });
});
