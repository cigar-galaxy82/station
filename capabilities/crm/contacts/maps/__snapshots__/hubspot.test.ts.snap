// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`crm/contacts/hubspot Create should perform successfully 1`] = `
Object {
  "value": Object {
    "id": "351",
  },
}
`;
